moffi_sdk/auth.py

```diff
@@ -3,43 +3,90 @@
 """
 
 from typing import Any, Dict
+from urllib.parse import urlencode
 
 import requests
+from requests.structures import CaseInsensitiveDict
 
-from moffi_sdk.exceptions import AuthenticationException
-from moffi_sdk.utils import MOFFI_API
+from moffi_sdk.exceptions import AuthenticationException, RequestException
 
 
-def signin(username: str, password: str) -> Dict[str, Any]:
-    """
-    Authenticate to Moffi API and return all profile informations
+class Session:
+    """Moffi session auth"""
 
-    Raise AuthenticationException in case of error
-    """
+    TOKEN: str
```

Comment:
`moffi_sdk/auth.py:37:8: C0103: Attribute name "TOKEN" doesn't conform to snake_case naming style (invalid-name)`